core/src/display_object/container.rs

```diff
@@ -843,6 +844,14 @@ impl<'gc> ChildContainer<'gc> {
         self.mouse_children = mouse_children;
     }
 
+    pub fn movie(&self) -> Arc<SwfMovie> {
+        self.movie.clone()
+    }
+
+    pub fn set_movie(&mut self, movie: Arc<SwfMovie>) {
```

Comment:
Is this unused?

Reply:
https://github.com/Lord-McSweeney/ruffle/blob/67cfc0e0dbff6c48b4fe2b13447ba715281848f6/core/src/display_object/stage.rs#L235